Python/compile.c

```diff
@@ -1519,6 +1553,17 @@ compiler_addop_j(struct compiler *c, int opcode, basicblock *b, int absolute)
     } \
 }
 
+/* These macros allows to check only for errors and not emmit bytecode
+ * while visiting nodes.
+*/
+
+#define BEGIN_DO_NOT_EMIT_BYTECODE { \
+    c->c_do_not_emit_bytecode++;
+
+#define END_DO_NOT_EMIT_BYTECODE \
+    c->c_do_not_emit_bytecode--; \
+}
```

Comment:
I don't understand the purpose of these macros. What is the problem with modying directly c_do_not_emit_bytecode? I dislike macros which expect variables with a specific name ("c"). I suggest to remove these macros.